examples/skybox/skybox_vert.glsl

```diff
@@ -0,0 +1,19 @@
+#version 450
+
+layout(location = 0) in vec2 position;
```

Comment:
we don't actually need vertex attributes for shaders that just draw a full-screen quad, instead we can generate the positions based on the vertex ID, like the other examples do, like the mipmap one:
```glsl
    vec2 tc = vec2(0.0);
    switch(gl_VertexIndex) {
        case 0: tc = vec2(1.0, 0.0); break;
        case 1: tc = vec2(1.0, 1.0); break;
        case 2: tc = vec2(0.0, 0.0); break;
        case 3: tc = vec2(0.0, 1.0); break;
    }
    v_TexCoord = tc;
    gl_Position = vec4(tc * 2.0 - 1.0, 0.5, 1.0);
```

Reply:
Ok this part was also a little unclear to me, so:

1. I have to send something to fire the shader, yes ? So just a Float for vertex type and pass 3 dummy values ?
2. What specific values should I set ? Same as the ones which I send in now in the rust part?

Reply:
Doing `draw()` is what fires the shader, given that there is a pipeline set. Nothing else is required.
2) yes, same ones are fine (for 3-vertex triangle)